lib/cdo/web_purify.rb

```diff
@@ -27,7 +27,7 @@ def self.find_potential_profanities(text, language_codes = ['en'])
       "&text=#{URI.encode(text)}" \
       "&lang=#{language_codes}" \
       "&format=json"
-    result = JSON.parse(open(url).read)
+    result = JSON.parse(open(url, open_timeout: 5, read_timeout: 10).read)
```

Comment:
Should we store these values in DCDO?

Reply:
yeah, i think so. a 10-second read timeout is pretty safe, but if you do end up needing to tweak it, it's nice to have that option available immediately, especially for something we use a lot like Webpurify

Reply:
That makes sense. I've added that. Let me know if I went to far with breaking this out into multiple, indented lines :)